lib/job-iteration/log_subscriber.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module JobIteration
+  class LogSubscriber < ActiveSupport::LogSubscriber
+    def logger
+      JobIteration.logger
+    end
+
+    def nil_enumerator(event)
+      info do
+        "[JobIteration::Iteration] `build_enumerator` returned nil. Skipping the job."
+      end
```

Comment:
Why pass a block here, instead of the string itself?
```suggestion
      info("[JobIteration::Iteration] `build_enumerator` returned nil. Skipping the job.")
```

Reply:
Ah, right. I was thinking about the overhead of calling into the block when you're logging it, but didn't consider the string interpolation cost when not logging it. I guess it comes down to optimizing for log levels above or below `info`, and how much difference it makes in those cases. Either way, I'm not too bothered - I'm fine keeping the blocks if you think that makes the most sense.